src/bundle/Resources/views/themes/admin/ui/edit_base.html.twig

```diff
@@ -4,10 +4,6 @@
 
 {% block header_row %}{% endblock %}
 
-{% block left_sidebar %}
```

Comment:
This is integral part of this view and should stay here even it it is empty.

Reply:
Otherwise, when extending this view without overwriting this block, the view will be broken.

Reply:
You are right. Fixed in 86b5f74